packages/sdk-middleware-auth/src/base-auth-flow.js

```diff
@@ -106,10 +115,18 @@ export default function authMiddlewareBase(
     pendingTasks,
     requestState,
     tokenCache,
+    fetch: fetcher,
   }: AuthMiddlewareBaseOptions,
   next: Next,
   userOptions?: AuthMiddlewareOptions | PasswordAuthMiddlewareOptions
 ) {
+  if (!fetcher && typeof fetch === 'undefined')
+    throw new Error(
+      '`fetch` is not available. Please pass in `fetch` as an option or have it globally available.'
+    )
+  if (!fetcher)
```

Comment:
I am open to suggestions here. I stole this from apollo-link-http assuming that they know better than we.

Reply:
I think it's fine yeah 👍